examples/spring-boot-on-kubernetes-without-annotations/readme.md

```diff
@@ -0,0 +1,95 @@
+#Example:  Add configmap data to a volume without annotations in Spring Boot Applications 
+
+An example that showcases how to generate a kubernetes resource utilizing Spring Boot-specific metadata. 
+To customize the generated manifests you need to add dekorate properties to your `application.yml` or `application.properties` descriptors. 
+and have the following dependency in your class path:
+
+    <dependency>
+          <groupId>io.dekorate</groupId>
+          <artifactId>dekorate-spring-boot</artifactId>
+          <version>${project.version}</version>
+    </dependency>
+
+The following annotation configuration:
+
+```
+@KubernetesApplication(group = "hello-world", labels = @Label(key = "foo", value = "bar"),
+  ports = @Port(name = "http", containerPort = 8080),
+  serviceType = ServiceType.LoadBalancer,
+  configMapVolumes = @ConfigMapVolume(volumeName = "bar-volume", configMapName = "foo-map"))
+@SpringBootApplication
+public class Main {
+
+  public static void main(String[] args) {
+    SpringApplication.run(Main.class, args);
+  }
+}
+```
+Can be expressed using properties:
```

Comment:
This text should be `can be expressed using dekorate kubernetes properties (if you use an application.properties file):`

Reply:
ok, done